api/api.go

```diff
@@ -8,16 +8,72 @@ import (
 	"strings"
 
 	"github.com/go-resty/resty/v2"
+	"github.com/mitchellh/mapstructure"
 )
 
-const splitioAPIUri = "https://sdk.split.io/api"
+const (
+	splitioAPIUri        = "https://sdk.split.io/api"
+	firstRequestSince    = -1
+	defaultMaxRequestNum = 100
+)
 
 // SplitioAPIBinding contains splitioAPIKey
 type SplitioAPIBinding struct {
 	splitioAPIKey string
 	splitioAPIUri string
 }
 
+// splitResponse contains the response for splitChange end point
+type splitResponse struct {
+	Splits []Split
+	Since  int
+	Till   int
+}
+
+// Split struct contains info for a split
+type Split struct {
```

Comment:
Are these copy pasted from the split SDK?  can we re-use the definitions from that module instead of copy pasting them?

Reply:
I actually created them by myself. 
but thanks for pointing it out, @rohanmurthy helps me to find that they have something I probably can directly use. I will re-write the code to directly use from their sdk.